### ISSUE TITLE
Changelog displayed when new Fastlane version is available pollutes the log

### ISSUE THREAD

**Issue by andriichernenko** [fastlane/fastlane#11604]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

When new version is available, Fastlane displays the changelog on every run. Updates are released quite often (which is great! I appreciate this a lot!), but unfortunately updating Fastlane modifies Gemfile.lock which is under version control. Updates involve bureaucracy (I need to create pull request, have it reviewed by my colleagues), so we don't do this for every new version (especially since some of them are released the same day, like 2.76.1 and 2.76.0).

In practice this means that way more often than not when using Fastlane I see the changelog. Description of changes between versions 2.75.1 released 9 days ago and 2.76.1 released 2 days ago together with other related messages occupies 34 lines of text displayed at the end of output, so if an invocation fails for some reason, I have to scroll up and scan all this text to find the error that caused the failure. Colors make this easier, but I still feel this is annoying and unnecessary. A minor annoyance, of course, but repeated many times. Death by a thousand cuts, if you will.

Perhaps Fastlane could be more discreet about the changelog?

**Comment by taquitos** [comment]
Maybe adding an option to provide a minimal-sized log output would be best, instead of listing all the changes. That way you still know you should update, but don't have to sift through reams of text.

**Comment by amlcurran** [comment]
As this is a "you can do this" task, I'd like to take a look at implementing it - I also find it someone distracting when debugging an error.

I see there is some [similar functionality](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/update_checker/update_checker.rb#L77) already there which prevents the changelog being displayed, so I will start from there!

**Comment by joshdholtz** [comment]
Just to follow up on @amlcurran's comment... The way to hide the change log is `FASTLANE_HIDE_CHANGELOG=true`. Just wanted to make sure that solution was easily readable for anybody else coming to this issue 😊 

<img width="540" alt="screen shot 2018-03-13 at 6 13 05 am" src="https://user-images.githubusercontent.com/401294/37338511-a79d564c-2685-11e8-9da0-153e239e3161.png">

@andriichernenko Is this what you were looking for? ^

**Comment by andriichernenko** [comment]
@joshdholtz What I had in mind was a message that occupies just one-two lines, something like:
```
fastlane 2.85.0 is available, you are on 2.83.0. Update using 'bundle update fastlane'. 
To see what's new, open https://github.com/fastlane/fastlane/releases
```
However, this already looks much better, maybe even good enough!

**Comment by joshdholtz** [comment]
@andriichernenko And its already build it 😊 Going to close this for now but lemme know if anything else comes up! Happy fastlaning 🚀
